### ISSUE TITLE
Unable to call Appwrite version without calling init

### ISSUE THREAD

**Issue by IwateKyle** [appwrite/appwrite#1255]
## 🐛 Bug Report

Following the instruction for installing the CLI - I came to the step where the instruction said to run the shell command
``` bash
appwrite version
``` 

then to call 

``` bash
appwrite init
``` 

However, the CLI complained after calling version stating that I hadn't initialized my app yet.

After satisfying the CLI by initializing an app - I was then able to get the version from the shell which is

```bash
CLI Version : 0.10.0
Server Version : 0.8.0
```

## Expected behavior

I was expecting that the CLI would have no requirements to show the version.

**Comment by christyjacob4** [comment]
@IwateKyle  Thank you for bringing this to our notice. We will fix this ASAP :slightly_smiling_face:

**Comment by christyjacob4** [comment]
@IwateKyle  This PR should fix this :smiley: 
https://github.com/appwrite/sdk-generator/pull/216

**Comment by eldadfux** [comment]
@christyjacob4 PR was merged, would you like to take care of the release?

**Comment by eldadfux** [comment]
This is now fixed and available as part of Appwrite 0.9:
https://dev.to/appwrite/announcing-appwrite-0-9-the-open-source-firebase-alternative-53ho 🥳 🎉
